invokeai/frontend/web/.eslintrc.js

```diff
@@ -122,6 +122,12 @@ module.exports = {
         'i18next/no-literal-string': 'off',
       },
     },
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      rules: {
+        'react/prop-types': 'off',
+      },
+    },
```

Comment:
Uhh this should not be necessary

Reply:
Without it there's a prop validation error so 🤷‍♂️

Reply:
It failed bc this is not correct:

```ts
memo<ImageSettingsAccordionProps>(({ collapseAdvanced = false }) => { ... })
```

memo's type argument is a component, not the props of a component.

It should be:
```ts
memo(({ collapseAdvanced = false }: ImageSettingsAccordionProps) => { ... })
```